docs/src/tutorials/implementing.md

```diff
@@ -276,36 +280,16 @@ For each attribute
     attribute. You should make sure that your [`get`](@ref) function correctly
     infers to this type (or a subtype of it).
 
-Each column in the table indicates whether you need to implement the particular
-method for each attribute.
+All `Optimizer`s must implement [`get`](@ref) for the the following attributes:
```

Comment:
This is the readable list, isn't it ? I would just say that the optimizers that want to be readable should implement the readable list detailed in the other sections ?